src/bosh_aws_cpi/lib/cloud/aws/instance_manager.rb

```diff
@@ -34,6 +34,7 @@ def create(agent_id, stemcell_id, resource_pool, networks_spec, disk_locality, e
         instance.wait_for_running
         instance.attach_to_load_balancers(resource_pool['elbs'] || [])
         instance.update_routing_tables(resource_pool['advertised_routes'] || [])
+        instance.source_dest_check = resource_pool['source_dest_check'] || true
```

Comment:
@fearoffish afraid you can't use the fallback operator here as `false || true` always evaluate to `true`. Your unit tests back this up as you set `'source_dest_check' => false` at the top, yet every test expects `allow(aws_instance).to receive(:source_dest_check=).with(true)`. Could you fix this and add tests for both the `false` and `true` cases?